docs/src/how-to/5-contribute-to-koma.md

```diff
@@ -62,13 +59,29 @@ The Julia extension should automatically detect the `KomaMRI` environment. To ch
 
 ### 5. KomaMRI monorepo setup
 
-As KomaMRI.jl contains multiple packages in one GitHub repository, you need to specify that you want to use your local copies (instead of the ones available on the Julia registries) with:
-```shell
-(KomaMRI) pkg> dev ./KomaMRIBase ./KomaMRICore ./KomaMRIFiles ./KomaMRIPlots
+As KomaMRI.jl contains multiple packages in one GitHub repository, you need to specify that you want to use your local copies (instead of the ones available on the Julia registries) and using the `instantiate` command to install all the required packages (specified in `project.toml`) with the following script:
+
+```julia-repl
+using Pkg
+Pkg.activate("KomaMRICore")
+Pkg.develop(path = "./KomaMRIBase")
+
+Pkg.activate("KomaMRIFiles")
+Pkg.develop(path = "./KomaMRIBase")
+
+Pkg.activate("KomaMRIPlots")
+Pkg.develop(path = "./KomaMRIBase")
+
+Pkg.activate(".")
+Pkg.develop(path = "./KomaMRICore")
+Pkg.develop(path = "./KomaMRIFiles")
+Pkg.develop(path = "./KomaMRIPlots")
 ```
-Finally, use the `instantiate` command to install all the required packages (specified in the `Project.toml`):
-```shell
-(KomaMRI) pkg> instantiate
+In case you want to contribute specifically in documentation, you will need to use the `docs` enviroment with the following script:
+
+```julia-repl
+Pkg.activate("docs")
+Pkg.develop(path = ".")
```

Comment:
- Do not use julia-repl syntax highlighting, use `julia`.

- Project.toml instead of project.toml.

- Are you sure at least one `instantiate` is necessary?

- You did not put the foldable part, as asked in a previous review comment https://github.com/JuliaHealth/KomaMRI.jl/pull/510#discussion_r1854757710. You could also do this to make it shorter:
```julia
using Pkg
# Koma sub-packages dev setup
koma_subpkgs = ["KomaMRICore", "KomaMRIFiles", "KomaMRIPlots"]
for pkg in koma_subpkgs
    Pkg.activate(pkg)
    Pkg.develop(path = "./KomaMRIBase")
end
# Main package (KomaMRI) dev setup
Pkg.activate(".")
for pkg in koma_subpkgs
    Pkg.develop(path = "./$pkg")
end
```

Reply:
fixed all points commented in commit [72b8a51](https://github.com/JuliaHealth/KomaMRI.jl/pull/510/commits/72b8a51175125a292cbf53ff7d7c5ac99f060bfb)